src/utils/positionHelper.ts

```diff
@@ -0,0 +1,96 @@
+import * as fs from 'fs';
+import { Cinema, acPosition } from '../types';
+import fetch from 'cross-fetch'
+
+interface OpenDataFranceReply {
+    lat:         number;
+    lon:         number;
+    citycode:    string;
+    city:        string;
+    name:        string;
+    postcode:    string;
+    housenumber: string;
+}
+
+let positions: acPosition[] = require('./cinePositions.json');
+let cinemas: { [id: string]: Cinema } = require('../../cinemas.json');
+
+export async function getPositionForCine(cine: Cinema): Promise<acPosition> {
+    let pos: acPosition = positions.find((_cine: any) => _cine.address === cine.address )
```

Comment:
might be safer to use the cine ID somehow, what do you think ?

Reply:
yes, I should add it in the sync function